src/inc/sift-events/sift-events.php

```diff
@@ -581,18 +575,11 @@ public static function update_or_create_order( string $order_id, \WC_Order $orde
 
 		$event = $create_order ? Sift_Event_Types::$create_order : Sift_Event_Types::$update_order;
 
-		// Log the event type we're trying to send for debugging.
-		Sift_For_WooCommerce::log(
-			sprintf( 'Preparing to send %s event for order %s', $event, $order_id ),
-			'debug',
-			array( 'source' => 'sift-order-events' )
-		);
-
 		if ( ! Sift_Event_Types::can_event_be_sent( $event ) ) {
 			// Log when event can't be sent due to settings.
 			Sift_For_WooCommerce::log(
```

Comment:
@bor0 I am pretty sure this is the issue.

You can remove this log: this is not an error but a normal behaviour: the event was cancelled by the settings.

Reply:
Nice! The more unnecessary log removals, the better ;)